src/target/renesas.c

```diff
@@ -948,3 +992,100 @@ static bool renesas_uid(target_s *t, int argc, const char **argv)
 
 	return true;
 }
+
+static bool renesas_print_cmd_usage(target_s *t)
+{
+	tc_printf(t, "command help goes here\n");
+	return false;
+}
+
+static bool renesas_cmd_option(target_s *t, int argc, const char **argv)
+{
+	(void)argc;
+	(void)argv;
+
+	const uint8_t arglen = strlen(argv[1]);
+
+	if (arglen > 0 && strncmp(argv[1], "get", arglen) == 0) {
+		if (argc != 3)
+			return renesas_print_cmd_usage(t);
+
+		target_addr_t option_addr = strtoul(argv[2], NULL, 0);
+		if (!renesas_check_option_setting(t, option_addr))
+			return renesas_print_cmd_usage(t);
+
+		/* The address must be aligned */
+		if (option_addr % 0x10 > 0)
+			return renesas_print_cmd_usage(t);
+
+		uint32_t value = target_mem_read32(t, option_addr);
+
+		tc_printf(t, "Read from address: 0x%02x, value: %" PRIx32 "\n", option_addr, value);
+
+		return true;
+	} else if (arglen > 0 && strncmp(argv[1], "set", arglen) == 0) {
+		if (argc != 4)
+			return renesas_print_cmd_usage(t);
+
+		target_addr_t option_addr = strtoul(argv[2], NULL, 0);
+		if (!renesas_check_option_setting(t, option_addr))
+			return renesas_print_cmd_usage(t);
+
+		/* The address must be aligned */
+		if (option_addr % 0x10 > 0)
+			return renesas_print_cmd_usage(t);
+
+		/* Configuration-Set command always needs 16 bytes of data */
+		const uint8_t len = 16UL;
+		uint8_t data[len];
+
+		for (size_t i = 0; i < len;) {
```

Comment:
`i += 4U` should be a part of this loop statement, not in the body of the loop. All constants used here should have the L suffix dropped especially as that might not even result in the right integer type (like in the uint8_t assignment above, where `-Wconversion` would warn about it truncating):

```suggestion
		for (size_t i = 0; i < ARRAY_LENGTH(data); i += 4U) {
```